Comparison Engine/Comparison Engine/Base Classes/Bar.cs

```diff
@@ -20,13 +20,37 @@ public Bar(int barID, string barName, string location)
             this.location = location;
         }
 
+        //Changes this bar object's name
+        public void EditBarName(string newName)
+        {
+            barName = newName;
+        }
+
+        //Changes this bar object's location
+        public void EditBarLocation(string newLocation)
+        {
+            location = newLocation;
+        }
```

Comment:
Only comment on this file would be if it's "barName", then it should be "barLocation" and if it's "location" then it would be more appropriate to have "name"